.github/workflows/ci.yml

```diff
@@ -208,13 +212,17 @@ jobs:
         run: npm run test-types
       - name: Run all tests (for non-Linux)
         if: ${{ matrix.os != 'ubuntu-latest' }}
-        run: npm run test -- --test-suite ${{ matrix.suite }}
+        run: npm run test -- --test-suite ${{ matrix.suite }} --save-stats-to /tmp/artifacts/${{ github.event_name }}_${{ github.job }}_${{ github.run_id }}.json
       - name: Install linux dependencies.
         if: ${{ matrix.os == 'ubuntu-latest' }}
         run: sudo apt-get install xvfb
       - name: Run all tests (for Linux)
         if: ${{ matrix.os == 'ubuntu-latest' }}
-        run: xvfb-run --auto-servernum npm run test -- --test-suite ${{ matrix.suite }}
+        run: xvfb-run --auto-servernum npm run test -- --test-suite ${{ matrix.suite }} --save-stats-to /tmp/artifacts/${{ github.event_name }}_${{ github.job }}_${{ github.run_id }}.json
```

Comment:
How to artifacts work? Do we need an unique name for each file? If so we this would be overridden on every matrix run.

Reply:
we are going to see how it works

Reply:
I expect that artifacts are scoped per run, at least they are shown so in the UI

Reply:
yeah needs some sort of id generation